pylint/checkers/base/comparison_checker.py

```diff
@@ -58,6 +59,13 @@ class ComparisonChecker(_BasicChecker):
             "comparison-with-itself",
             "Used when something is compared against itself.",
         ),
+        "R0125": (
```

Comment:
```suggestion
        "R0133": (
```

We actually already emit `W0125`. `0133` is free though!

Reply:
Is there a catalogue of free and used numbers?

Reply:
Sadly, no...

Numbers are made up of a prefix (the first two) and an "id" (the last two digits). Sadly, previously we have not been very strict on not re-using numbers, so for prefixes 01-10 we have a lot of duplicate numbers. For 11+ we do check for duplicates, but due to large amount of duplicates for those first ten prefixes we haven't. (At some point I might make a check to also do that, but haven't gotten around making a full list yet).

Thus, for the lower prefixes the only thing you can do is `grep` or search for 0125, 0126, 0127... Luckily such number combinations aren't really used anywhere else than in message id's so it's a relatively easy process.